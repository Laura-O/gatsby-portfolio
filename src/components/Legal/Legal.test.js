import React from 'react';
import { shallow } from 'enzyme';
import Legal from './Legal';

test('rendering', () => {
  const wrapper = shallow(<Legal />);
  expect(wrapper).toMatchSnapshot();
});
