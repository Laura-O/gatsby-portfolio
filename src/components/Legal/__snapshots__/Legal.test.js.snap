// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<div
  className="credits"
>
  <div>
    <h3>
      Credits
    </h3>
    Icons made by
     
    <a
      href="https://www.flaticon.com/authors/smashicons"
    >
      Smashicons
    </a>
     and
     
    <a
      href="https://www.flaticon.com/authors/swifticons"
    >
      Swifticons
    </a>
    from 
    <a
      href="https://www.flaticon.com"
    >
      www.flaticon.com
    </a>
  </div>
</div>
`;
